types/ember-changeset-validations/tslint.json

```
@@ -0,0 +1,7 @@
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
+        "npm-naming": false
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
the package has proper registration, what's the reason of those exclusions?

---

I looked more around the packages, I can specify npm-naming more for my default export issue without overriding it completely, like that `"npm-naming": [true, { "mode": "code", "errors": [["NoDefaultExport", false]] }]`
I think it's better